examples/js/loaders/FBXLoader.js

```diff
@@ -2007,9 +2007,7 @@
 			var preRotations = new THREE.Euler().fromArray( array );
 
 			preRotations = new THREE.Quaternion().setFromEuler( preRotations );
-			var currentRotation = new THREE.Quaternion().setFromEuler( model.rotation );
-			preRotations.multiply( currentRotation );
-			model.rotation.setFromQuaternion( preRotations, 'ZYX' );
+			model.quaternion.premultiply( preRotations );
```

Comment:
@takahirox @looeee I think this is equivalent to what was intended...

Reply:
Yep, this looks good too 🙂